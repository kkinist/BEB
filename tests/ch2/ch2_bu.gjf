%chk=ch2.chk
%nprocs=4
%mem=1000mw
# HF/GEN gfinput geom=check IOP(6/81=3) scf=xqc pop(all,thresh=1)

BEB step 2: Hartree-Fock B and U for ch2

0 3

C H 0
6-311G(d,p)
****

