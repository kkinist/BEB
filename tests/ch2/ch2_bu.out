 Entering Gaussian System, Link 0=g09
 Initial command:
 /usr/local/apps/g09/l1.exe "/wrk/irikura/Gau-26899.inp" -scrdir="/wrk/irikura/"
 Entering Link 1 = /usr/local/apps/g09/l1.exe PID=     26901.
  
 Copyright (c) 1988,1990,1992,1993,1995,1998,2003,2009,2013,
            Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 09 program.  It is based on
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 09, Revision D.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, B. Mennucci, 
 G. A. Petersson, H. Nakatsuji, M. Caricato, X. Li, H. P. Hratchian, 
 A. F. Izmaylov, J. Bloino, G. Zheng, J. L. Sonnenberg, M. Hada, 
 M. Ehara, K. Toyota, R. Fukuda, J. Hasegawa, M. Ishida, T. Nakajima, 
 Y. Honda, O. Kitao, H. Nakai, T. Vreven, J. A. Montgomery, Jr., 
 J. E. Peralta, F. Ogliaro, M. Bearpark, J. J. Heyd, E. Brothers, 
 K. N. Kudin, V. N. Staroverov, T. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. Rendell, J. C. Burant, S. S. Iyengar, J. Tomasi, 
 M. Cossi, N. Rega, J. M. Millam, M. Klene, J. E. Knox, J. B. Cross, 
 V. Bakken, C. Adamo, J. Jaramillo, R. Gomperts, R. E. Stratmann, 
 O. Yazyev, A. J. Austin, R. Cammi, C. Pomelli, J. W. Ochterski, 
 R. L. Martin, K. Morokuma, V. G. Zakrzewski, G. A. Voth, 
 P. Salvador, J. J. Dannenberg, S. Dapprich, A. D. Daniels, 
 O. Farkas, J. B. Foresman, J. V. Ortiz, J. Cioslowski, 
 and D. J. Fox, Gaussian, Inc., Wallingford CT, 2013.
 
 ******************************************
 Gaussian 09:  ES64L-G09RevD.01 24-Apr-2013
                17-May-2017 
 ******************************************
 %chk=ch2.chk
 %nprocs=4
 Will use up to    4 processors via shared memory.
 %mem=1000mw
 -----------------------------------------------------------------
 # HF/GEN gfinput geom=check IOP(6/81=3) scf=xqc pop(all,thresh=1)
 -----------------------------------------------------------------
 1/29=2,38=1/1;
 2/12=2,40=1/2;
 3/5=7,11=9,14=-4,16=1,24=10,25=1,30=1/1,2,3;
 4//1;
 5/5=2,8=3,13=1,38=5/2,8;
 6/7=2,8=2,9=2,10=2,28=1,81=3,87=-1,88=1/1;
 99/5=1,9=1/99;
 ----------------------------------------
 BEB step 2: Hartree-Fock B and U for ch2
 ----------------------------------------
 Structure from the checkpoint file:  "ch2.chk"
 Charge =  0 Multiplicity = 3
 Redundant internal coordinates found in file.
 C,0,0.,0.,0.0661340759
 H,0,0.,-0.9932409277,-0.3639822479
 H,0,0.,0.9932409277,-0.3639822479
 Recover connectivity data from disk.
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        0.000000    0.000000    0.066134
      2          1           0        0.000000   -0.993241   -0.363982
      3          1           0        0.000000    0.993241   -0.363982
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3
     1  C    0.000000
     2  H    1.082371   0.000000
     3  H    1.082371   1.986482   0.000000
 Stoichiometry    CH2(3)
 Framework group  C2V[C2(C),SGV(H2)]
 Deg. of freedom     2
 Full point group                 C2V     NOp   4
 Largest Abelian subgroup         C2V     NOp   4
 Largest concise Abelian subgroup C2      NOp   2
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        0.000000    0.000000    0.107529
      2          1           0        0.000000    0.993241   -0.322587
      3          1           0        0.000000   -0.993241   -0.322587
 ---------------------------------------------------------------------
 Rotational constants (GHZ):   1582.9318105    254.1515972    218.9909540
 General basis read from cards:  (5D, 7F)
 AO basis set in the form of general basis input (Overlap normalization):
      1 0
 S   6 1.00       0.000000000000
      0.4563240000D+04  0.1966650249D-02
      0.6820240000D+03  0.1523060193D-01
      0.1549730000D+03  0.7612690966D-01
      0.4445530000D+02  0.2608010331D+00
      0.1302900000D+02  0.6164620782D+00
      0.1827730000D+01  0.2210060280D+00
 SP   3 1.00       0.000000000000
      0.2096420000D+02  0.1146600807D+00  0.4024869267D-01
      0.4803310000D+01  0.9199996477D+00  0.2375939567D+00
      0.1459330000D+01 -0.3030682134D-02  0.8158538515D+00
 SP   1 1.00       0.000000000000
      0.4834560000D+00  0.1000000000D+01  0.1000000000D+01
 SP   1 1.00       0.000000000000
      0.1455850000D+00  0.1000000000D+01  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.6260000000D+00  0.1000000000D+01
 ****
      2 0
 S   3 1.00       0.000000000000
      0.3386500000D+02  0.2549381454D-01
      0.5094790000D+01  0.1903731086D+00
      0.1158790000D+01  0.8521614860D+00
 S   1 1.00       0.000000000000
      0.3258400000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027410000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.7500000000D+00  0.1000000000D+01
 ****
      3 0
 S   3 1.00       0.000000000000
      0.3386500000D+02  0.2549381454D-01
      0.5094790000D+01  0.1903731086D+00
      0.1158790000D+01  0.8521614860D+00
 S   1 1.00       0.000000000000
      0.3258400000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027410000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.7500000000D+00  0.1000000000D+01
 ****

 There are    15 symmetry adapted cartesian basis functions of A1  symmetry.
 There are     2 symmetry adapted cartesian basis functions of A2  symmetry.
 There are     5 symmetry adapted cartesian basis functions of B1  symmetry.
 There are     9 symmetry adapted cartesian basis functions of B2  symmetry.
 There are    14 symmetry adapted basis functions of A1  symmetry.
 There are     2 symmetry adapted basis functions of A2  symmetry.
 There are     5 symmetry adapted basis functions of B1  symmetry.
 There are     9 symmetry adapted basis functions of B2  symmetry.
    30 basis functions,    48 primitive gaussians,    31 cartesian basis functions
     5 alpha electrons        3 beta electrons
       nuclear repulsion energy         6.1332544348 Hartrees.
 NAtoms=    3 NActive=    3 NUniq=    2 SFac= 2.25D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=    30 RedAO= T EigKep=  2.91D-02  NBF=    14     2     5     9
 NBsUse=    30 1.00D-06 EigRej= -1.00D+00 NBFU=    14     2     5     9
 ExpMin= 1.03D-01 ExpMax= 4.56D+03 ExpMxC= 6.82D+02 IAcc=1 IRadAn=         1 AccDes= 0.00D+00
 Harris functional with IExCor=  205 and IRadAn=       1 diagonalized for initial guess.
 HarFok:  IExCor=  205 AccDes= 0.00D+00 IRadAn=         1 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=     500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Initial guess orbital symmetries:
 Alpha Orbitals:
       Occupied  (A1) (A1) (B2) (A1) (B1)
       Virtual   (A1) (B2) (A1) (B2) (B1) (A1) (B2) (A1) (A2) (A1)
                 (B1) (B2) (A1) (B1) (B2) (A1) (A2) (B2) (B2) (A1)
                 (A1) (B1) (A1) (B2) (A1)
 Beta  Orbitals:
       Occupied  (A1) (A1) (B2)
       Virtual   (A1) (B1) (A1) (B2) (A1) (B2) (B1) (A1) (B2) (A1)
                 (A2) (A1) (B1) (B2) (A1) (B1) (B2) (A1) (A2) (B2)
                 (B2) (A1) (A1) (B1) (A1) (B2) (A1)
 The electronic state of the initial guess is 3-B1.
 Initial guess <Sx>= 0.0000 <Sy>= 0.0000 <Sz>= 1.0000 <S**2>= 2.0000 S= 1.0000
 Keep R1 and R2 ints in memory in symmetry-blocked form, NReq=949519.
 Requested convergence on RMS density matrix=1.00D-08 within  64 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(UHF) =  -38.9338846432     A.U. after   12 cycles
            NFock= 12  Conv=0.30D-08     -V/T= 2.0007
 <Sx>= 0.0000 <Sy>= 0.0000 <Sz>= 1.0000 <S**2>= 2.0162 S= 1.0054
 <L.S>= 0.000000000000E+00
 Annihilation of the first spin contaminant:
 S**2 before annihilation     2.0162,   after     2.0001
 QCSCF skips out because SCF is already converged.

 **********************************************************************

            Population analysis using the SCF density.

 **********************************************************************

 Orbital symmetries:
 Alpha Orbitals:
       Occupied  (A1) (A1) (B2) (A1) (B1)
       Virtual   (A1) (B2) (A1) (B2) (B1) (A1) (B2) (A1) (A2) (A1)
                 (B1) (B2) (A1) (B1) (B2) (A1) (A2) (B2) (A1) (B2)
                 (A1) (B1) (A1) (B2) (A1)
 Beta  Orbitals:
       Occupied  (A1) (A1) (B2)
       Virtual   (A1) (B1) (A1) (B2) (A1) (B2) (B1) (A1) (B2) (A1)
                 (A2) (A1) (B1) (B2) (A1) (B1) (B2) (A1) (A2) (B2)
                 (A1) (B2) (A1) (B1) (A1) (B2) (A1)
 The electronic state is 3-B1.
 Alpha  occ. eigenvalues --  -11.26329  -0.94661  -0.61467  -0.45069  -0.40894
 Alpha virt. eigenvalues --    0.16620   0.21656   0.43302   0.49251   0.56412
 Alpha virt. eigenvalues --    0.57188   0.80434   0.84853   1.24294   1.40371
 Alpha virt. eigenvalues --    1.42938   1.54343   1.68949   1.95070   1.98688
 Alpha virt. eigenvalues --    2.10559   2.28668   2.57784   2.75711   2.82994
 Alpha virt. eigenvalues --    2.96688   3.00002   3.17217   4.03352  24.91598
  Beta  occ. eigenvalues --  -11.21472  -0.77364  -0.58274
  Beta virt. eigenvalues --    0.12672   0.15180   0.18647   0.22885   0.46907
  Beta virt. eigenvalues --    0.53198   0.72062   0.72376   0.84544   0.90712
  Beta virt. eigenvalues --    1.31109   1.51867   1.56914   1.56940   1.77823
  Beta virt. eigenvalues --    2.00018   2.03044   2.13346   2.34232   2.58752
  Beta virt. eigenvalues --    2.76740   2.85231   3.01849   3.10553   3.24642
  Beta virt. eigenvalues --    4.06555  24.95111
          Condensed to atoms (all electrons):
               1          2          3
     1  C    5.496176   0.380539   0.380539
     2  H    0.380539   0.516861  -0.026028
     3  H    0.380539  -0.026028   0.516861
          Atomic-Atomic Spin Densities.
               1          2          3
     1  C    2.270517  -0.029522  -0.029522
     2  H   -0.029522  -0.086358   0.010143
     3  H   -0.029522   0.010143  -0.086358
 Mulliken charges and spin densities:
               1          2
     1  C   -0.257255   2.211473
     2  H    0.128627  -0.105737
     3  H    0.128627  -0.105737
 Sum of Mulliken charges =   0.00000   2.00000
 Mulliken charges and spin densities with hydrogens summed into heavy atoms:
               1          2
     1  C    0.000000   2.000000
 Electronic spatial extent (au):  <R**2>=             24.0119
 Charge=              0.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              0.0000    Y=              0.0000    Z=             -0.5855  Tot=              0.5855
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             -8.0529   YY=             -5.8680   ZZ=             -7.5660
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             -0.8906   YY=              1.2943   ZZ=             -0.4037
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=              0.0000  YYY=              0.0000  ZZZ=             -0.9071  XYY=              0.0000
  XXY=              0.0000  XXZ=             -0.3081  XZZ=              0.0000  YZZ=              0.0000
  YYZ=             -0.9946  XYZ=              0.0000
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=             -9.1372 YYYY=            -13.1059 ZZZZ=             -9.8244 XXXY=              0.0000
 XXXZ=              0.0000 YYYX=              0.0000 YYYZ=              0.0000 ZZZX=              0.0000
 ZZZY=              0.0000 XXYY=             -4.2654 XXZZ=             -3.1742 YYZZ=             -3.9760
 XXYZ=              0.0000 YYXZ=              0.0000 ZZXY=              0.0000
 N-N= 6.133254434788D+00 E-N=-1.027873424102D+02  KE= 3.890837789267D+01
 Symmetry A1   KE= 3.579294515002D+01
 Symmetry A2   KE= 9.624544418146D-35
 Symmetry B1   KE= 1.166214428766D+00
 Symmetry B2   KE= 1.949218313882D+00

 Atomic contributions to Alpha molecular orbitals:
 Alpha occ 1 OE=-11.263 is C1-s=1.00
 Alpha occ 2 OE=-0.947 is C1-s=0.79 H2-s=0.10 H3-s=0.10
 Alpha occ 3 OE=-0.615 is C1-p=0.55 H3-s=0.22 H2-s=0.22
 Alpha occ 4 OE=-0.451 is C1-p=0.85 C1-s=0.05 H3-s=0.04 H2-s=0.04
 Alpha occ 5 OE=-0.409 is C1-p=0.99
 Alpha vir 6 OE=0.166 is H3-s=0.63 H2-s=0.63 C1-s=-0.28
 Alpha vir 7 OE=0.217 is H3-s=0.63 H2-s=0.63 C1-p=-0.28
 Alpha vir 8 OE=0.433 is H3-s=0.28 H2-s=0.28 C1-p=0.23 C1-s=0.22
 Alpha vir 9 OE=0.493 is C1-p=0.82 H3-s=0.09 H2-s=0.09 C1-d=-0.01
 Alpha vir 10 OE=0.564 is C1-p=1.00
 Alpha vir 11 OE=0.572 is C1-p=0.83 C1-s=0.12 H2-s=0.03 H3-s=0.03
 Alpha vir 12 OE=0.804 is C1-p=0.92 H3-s=0.02 H2-s=0.02 H2-p=0.02 H3-p=0.02 C1-d=0.01
 Alpha vir 13 OE=0.849 is C1-s=1.08 H3-s=-0.13 H2-s=-0.13 C1-p=0.07 H2-p=0.05 H3-p=0.05
 Alpha vir 14 OE=1.243 is C1-d=0.49 H2-p=0.26 H3-p=0.26
 Alpha vir 15 OE=1.404 is C1-d=0.69 H2-p=0.15 H3-p=0.15
 Alpha vir 16 OE=1.429 is C1-d=0.75 H2-p=0.13 H3-p=0.13
 Alpha vir 17 OE=1.543 is H3-p=0.32 H2-p=0.32 C1-d=0.31 H3-s=0.03 H2-s=0.03 C1-p=-0.02
 Alpha vir 18 OE=1.689 is C1-d=0.75 H2-s=0.08 H3-s=0.08 H2-p=0.06 H3-p=0.06 C1-s=-0.03 C1-p=-0.02
 Alpha vir 19 OE=1.951 is H2-p=0.37 H3-p=0.37 C1-d=0.25 C1-p=0.01
 Alpha vir 20 OE=1.987 is C1-d=0.47 H2-p=0.15 H3-p=0.15 H3-s=0.13 H2-s=0.13 C1-p=-0.04
 Alpha vir 21 OE=2.106 is C1-d=0.24 H2-p=0.24 H3-p=0.24 H2-s=0.13 H3-s=0.13 C1-p=0.04
 Alpha vir 22 OE=2.287 is C1-d=0.51 H2-p=0.24 H3-p=0.24
 Alpha vir 23 OE=2.578 is H2-s=0.30 H3-s=0.30 H2-p=0.24 H3-p=0.24 C1-d=-0.04 C1-p=-0.03
 Alpha vir 24 OE=2.757 is H3-p=0.35 H2-p=0.35 H3-s=0.20 H2-s=0.20 C1-d=-0.06 C1-s=-0.02 C1-p=-0.02
 Alpha vir 25 OE=2.830 is H2-p=0.27 H3-p=0.27 C1-p=0.26 C1-d=0.12 H2-s=0.04 H3-s=0.04
 Alpha vir 26 OE=2.967 is C1-p=0.53 C1-d=0.16 H3-p=0.09 H2-p=0.09 H2-s=0.04 H3-s=0.04 C1-s=0.03
 Alpha vir 27 OE=3.000 is C1-p=1.00
 Alpha vir 28 OE=3.172 is C1-p=0.47 C1-d=0.20 H3-s=0.09 H2-s=0.09 C1-s=0.05 H3-p=0.05 H2-p=0.05
 Alpha vir 29 OE=4.034 is C1-p=0.82 C1-d=0.13 H2-s=0.03 H3-s=0.03
 Alpha vir 30 OE=24.916 is C1-s=1.00

 Atomic contributions to Beta  molecular orbitals:
 Beta  occ 1 OE=-11.215 is C1-s=1.00
 Beta  occ 2 OE=-0.774 is C1-s=0.53 H2-s=0.21 H3-s=0.21 C1-p=0.03
 Beta  occ 3 OE=-0.583 is C1-p=0.44 H3-s=0.27 H2-s=0.27
 Beta  vir 4 OE=0.127 is C1-p=0.74 C1-s=0.21 H3-s=0.02 H2-s=0.02
 Beta  vir 5 OE=0.152 is C1-p=0.98
 Beta  vir 6 OE=0.186 is H2-s=0.59 H3-s=0.59 C1-s=-0.27 C1-p=0.08
 Beta  vir 7 OE=0.229 is H2-s=0.62 H3-s=0.62 C1-p=-0.26 H2-p=0.01 H3-p=0.01
 Beta  vir 8 OE=0.469 is C1-s=0.40 H2-s=0.24 H3-s=0.24 C1-p=0.15 C1-d=-0.02
 Beta  vir 9 OE=0.532 is C1-p=0.92 H3-s=0.05 H2-s=0.05 C1-d=-0.03
 Beta  vir 10 OE=0.721 is C1-p=0.99
 Beta  vir 11 OE=0.724 is C1-p=0.95 C1-s=0.03
 Beta  vir 12 OE=0.845 is C1-p=0.88 H3-s=0.03 H2-s=0.03 H2-p=0.02 H3-p=0.02
 Beta  vir 13 OE=0.907 is C1-s=1.09 H3-s=-0.12 H2-s=-0.12 H2-p=0.06 H3-p=0.06 C1-p=0.03
 Beta  vir 14 OE=1.311 is C1-d=0.43 H3-p=0.29 H2-p=0.29
 Beta  vir 15 OE=1.519 is C1-d=0.55 H2-p=0.21 H3-p=0.21 C1-p=0.02 H3-s=0.01 H2-s=0.01
 Beta  vir 16 OE=1.569 is C1-d=0.61 H2-p=0.19 H3-p=0.19 C1-p=0.02
 Beta  vir 17 OE=1.569 is H2-p=0.36 H3-p=0.36 C1-d=0.26 H2-s=0.02 H3-s=0.02
 Beta  vir 18 OE=1.778 is C1-d=0.82 H2-s=0.09 H3-s=0.09 C1-s=-0.04 H2-p=0.03 H3-p=0.03 C1-p=-0.01
 Beta  vir 19 OE=2.000 is C1-d=0.39 H2-p=0.30 H3-p=0.30 C1-p=0.01
 Beta  vir 20 OE=2.030 is C1-d=0.51 H2-s=0.16 H3-s=0.16 H2-p=0.11 H3-p=0.11 C1-p=-0.05
 Beta  vir 21 OE=2.133 is C1-d=0.30 H2-p=0.21 H3-p=0.21 H2-s=0.12 H3-s=0.12 C1-p=0.03
 Beta  vir 22 OE=2.342 is C1-d=0.57 H2-p=0.21 H3-p=0.21
 Beta  vir 23 OE=2.588 is H3-s=0.28 H2-s=0.28 H3-p=0.26 H2-p=0.26 C1-d=-0.04 C1-p=-0.03
 Beta  vir 24 OE=2.767 is H2-p=0.35 H3-p=0.35 H2-s=0.20 H3-s=0.20 C1-d=-0.06 C1-p=-0.02
 Beta  vir 25 OE=2.852 is C1-p=0.27 H2-p=0.25 H3-p=0.25 C1-d=0.15 H2-s=0.04 H3-s=0.04
 Beta  vir 26 OE=3.018 is C1-p=0.39 C1-d=0.24 H2-p=0.11 H3-p=0.11 H2-s=0.06 H3-s=0.06 C1-s=0.04
 Beta  vir 27 OE=3.106 is C1-p=1.00
 Beta  vir 28 OE=3.246 is C1-p=0.62 C1-d=0.17 H2-s=0.06 H3-s=0.06 C1-s=0.04 H3-p=0.03 H2-p=0.03
 Beta  vir 29 OE=4.066 is C1-p=0.83 C1-d=0.14 H2-s=0.03 H3-s=0.03 H2-p=-0.01 H3-p=-0.01
 Beta  vir 30 OE=24.951 is C1-s=1.00
 
 Orbital energies and kinetic energies (alpha):
                                 1                 2
   1         (A1)--O         -11.263292         15.997904
   2         (A1)--O          -0.946609          1.393532
   3         (B2)--O          -0.614673          1.002641
   4         (A1)--O          -0.450693          1.167526
   5         (B1)--O          -0.408943          1.166214
   6         (A1)--V           0.166200          0.479263
   7         (B2)--V           0.216562          0.444923
   8         (A1)--V           0.433015          0.858996
   9         (B2)--V           0.492505          0.748887
  10         (B1)--V           0.564116          1.304177
  11         (A1)--V           0.571884          1.378225
  12         (B2)--V           0.804336          1.869987
  13         (A1)--V           0.848527          2.362263
  14         (A2)--V           1.242940          1.817370
  15         (A1)--V           1.403706          2.018390
  16         (B1)--V           1.429375          2.055881
  17         (B2)--V           1.543428          2.038268
  18         (A1)--V           1.689493          2.356830
  19         (B1)--V           1.950700          2.461843
  20         (B2)--V           1.986877          2.696248
  21         (A1)--V           2.105587          2.885700
  22         (A2)--V           2.286675          2.800407
  23         (B2)--V           2.577839          3.482585
  24         (A1)--V           2.757111          3.727298
  25         (B2)--V           2.829937          4.903158
  26         (A1)--V           2.966876          5.436805
  27         (B1)--V           3.000019          6.286154
  28         (A1)--V           3.172172          5.417422
  29         (B2)--V           4.033520          6.346756
  30         (A1)--V          24.915978         60.377763
 Orbital energies and kinetic energies (beta):
                                 1                 2
   1         (A1)--O         -11.214722         16.058617
   2         (A1)--O          -0.773640          1.175366
   3         (B2)--O          -0.582743          0.946578
   4         (A1)--V           0.126722          0.732227
   5         (B1)--V           0.151797          0.670852
   6         (A1)--V           0.186467          0.471745
   7         (B2)--V           0.228849          0.442134
   8         (A1)--V           0.469067          0.881364
   9         (B2)--V           0.531981          0.716392
  10         (B1)--V           0.720620          1.704903
  11         (A1)--V           0.723760          1.738837
  12         (B2)--V           0.845444          1.944578
  13         (A1)--V           0.907124          2.478670
  14         (A2)--V           1.311092          1.803240
  15         (A1)--V           1.518666          2.020511
  16         (B1)--V           1.569141          2.048843
  17         (B2)--V           1.569404          2.025294
  18         (A1)--V           1.778232          2.394672
  19         (B1)--V           2.000179          2.475755
  20         (B2)--V           2.030437          2.756889
  21         (A1)--V           2.133455          2.875355
  22         (A2)--V           2.342325          2.814536
  23         (B2)--V           2.587517          3.418821
  24         (A1)--V           2.767402          3.741547
  25         (B2)--V           2.852307          4.922727
  26         (A1)--V           3.018492          5.202979
  27         (B1)--V           3.105528          6.373916
  28         (A1)--V           3.246422          5.713253
  29         (B2)--V           4.065554          6.360039
  30         (A1)--V          24.951111         60.372774
 Total kinetic energy from orbitals= 3.890837789267D+01
                          Isotropic Fermi Contact Couplings
        Atom                 a.u.       MegaHertz       Gauss      10(-4) cm-1
     1  C(13)              0.55377     311.27367     111.07018     103.82972
     2  H(1)              -0.02493     -55.72215     -19.88305     -18.58691
     3  H(1)              -0.02493     -55.72215     -19.88305     -18.58691
 --------------------------------------------------------
       Center         ----  Spin Dipole Couplings  ----
                      3XX-RR        3YY-RR        3ZZ-RR
 --------------------------------------------------------
     1   Atom        0.671556     -1.040208      0.368652
     2   Atom       -0.066810      0.121432     -0.054622
     3   Atom       -0.066810      0.121432     -0.054622
 --------------------------------------------------------
                        XY            XZ            YZ
 --------------------------------------------------------
     1   Atom        0.000000      0.000000      0.000000
     2   Atom        0.000000      0.000000     -0.090202
     3   Atom        0.000000      0.000000      0.090202
 --------------------------------------------------------


 ---------------------------------------------------------------------------------
              Anisotropic Spin Dipole Couplings in Principal Axis System
 ---------------------------------------------------------------------------------

       Atom             a.u.   MegaHertz   Gauss  10(-4) cm-1        Axes

              Baa    -1.0402  -139.586   -49.808   -46.561  0.0000  1.0000  0.0000
     1 C(13)  Bbb     0.3687    49.470    17.652    16.501  0.0000  0.0000  1.0000
              Bcc     0.6716    90.116    32.156    30.060  1.0000  0.0000  0.0000
 
              Baa    -0.0926   -49.424   -17.636   -16.486  0.0000  0.3883  0.9215
     2 H(1)   Bbb    -0.0668   -35.646   -12.720   -11.890  1.0000  0.0000  0.0000
              Bcc     0.1594    85.070    30.355    28.376  0.0000  0.9215 -0.3883
 
              Baa    -0.0926   -49.424   -17.636   -16.486  0.0000 -0.3883  0.9215
     3 H(1)   Bbb    -0.0668   -35.646   -12.720   -11.890  1.0000  0.0000  0.0000
              Bcc     0.1594    85.070    30.355    28.376  0.0000  0.9215  0.3883
 

 ---------------------------------------------------------------------------------


 Test job not archived.
 1\1\GINC-GAMBA\SP\UHF\Gen\C1H2(3)\IRIKURA\17-May-2017\0\\# HF/GEN gfin
 put geom=check IOP(6/81=3) scf=xqc pop(all,thresh=1)\\BEB step 2: Hart
 ree-Fock B and U for ch2\\0,3\C,0,0.,0.,0.0661340759\H,0,0.,-0.9932409
 277,-0.3639822479\H,0,0.,0.9932409277,-0.3639822479\\Version=ES64L-G09
 RevD.01\State=3-B1\HF=-38.9338846\S2=2.016205\S2-1=0.\S2A=2.000074\RMS
 D=2.995e-09\Dipole=0.,0.,-0.2303503\Quadrupole=-0.662128,0.9622824,-0.
 3001545,0.,0.,0.\PG=C02V [C2(C1),SGV(H2)]\\@


 THE PROGRESS OF RIVERS TO THE SEA IS NOT AS RAPID
 AS THAT OF MAN TO ERROR.
                                              -- VOLTAIRE
 Job cpu time:       0 days  0 hours  0 minutes  5.0 seconds.
 File lengths (MBytes):  RWF=      5 Int=      0 D2E=      0 Chk=      1 Scr=      1
 Normal termination of Gaussian 09 at Wed May 17 12:52:31 2017.
